internal/services/netapp/netapp_volume_resource.go

```diff
@@ -258,10 +264,19 @@ func resourceNetAppVolume() *pluginsdk.Resource {
 				},
 			},
 
-			"snapshot_directory_visible": {
-				Type:     pluginsdk.TypeBool,
+			"data_protection_snapshot_policy": {
+				Type:     pluginsdk.TypeList,
 				Optional: true,
-				Default:  true,
+				MaxItems: 1,
+				Elem: &pluginsdk.Resource{
+					Schema: map[string]*pluginsdk.Schema{
+						"snapshot_policy_id": {
```

Comment:
is there any reason to nest this in here instead of simply having `data_protection_snapshot_policy_id`

Reply:
are there current plans to include another property in this block?